lighthouse-core/audits/byte-efficiency/uses-responsive-images.js

```diff
@@ -82,11 +85,11 @@ class UsesResponsiveImages extends ByteEfficiencyAudit {
   }
 
   /**
-   * @param {LH.Artifacts.ImageElement & {naturalWidth: number, naturalHeight: number}} image
-   * @param {LH.Artifacts.ViewportDimensions} ViewportDimensions
-   * @param {Array<LH.Artifacts.NetworkRequest>} networkRecords
-   * @return {null|LH.Audit.ByteEfficiencyItem};
-   */
+    * @param {LH.Artifacts.ImageElement & {naturalWidth: number, naturalHeight: number}} image
```

Comment:
Can you check your editor autoformatting and un-modify stylistic changes to this file?

Reply:
Sure thing, should be fixed now.